<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registart</class>
 <widget class="QDialog" name="registart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{border-width:2px;
border-color:gray;
border-radius:10px;
padding:2px;
border-style:outset;
}
QPushButton:pressed{
background-color: rgb(91, 85, 255);
}

QLineEdit{
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:gray;
padding:3px;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="cardIdLabel">
        <property name="text">
         <string>身份证号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cardIdEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="querryBtn">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="roomIdLabel">
        <property name="text">
         <string> 房间号 ： </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="roomIdEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="reserInBtn">
        <property name="text">
         <string>登记入住</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
